0001-interledger-architecture/0001-interledger-architecture.md

```diff
@@ -1,252 +1,135 @@
 # Interledger Architecture
 
-This document outlines the Interledger architecture and explains how the different layers relate to each other. The [Interledger Protocol (ILP)](#interledger-protocol-ilp) is one layer in the Interledger architecture.
+The Interledger protocol suite provides for secure payments across multiple assets on different ledgers.
 
-The Interledger architecture is heavily inspired by the Internet architecture described in [RFC 1122](https://tools.ietf.org/html/rfc1122), [RFC 1123](https://tools.ietf.org/html/rfc1123) and [RFC 1009](https://tools.ietf.org/html/rfc1009).
+The [Interledger Protocol (ILP)](#interledger-layer) is the core of the Interledger protocol suite.
 
-## Introduction
+This document outlines the Interledger architecture and the functions of each of the layers of the stack.
 
-This document is intended to provide an overview over the Interledger architecture and protocol suite.
+The Interledger architecture is heavily inspired by the Internet architecture described in [RFC 1122](https://tools.ietf.org/html/rfc1122), [RFC 1123](https://tools.ietf.org/html/rfc1123) and [RFC 1009](https://tools.ietf.org/html/rfc1009).
 
-## The Interledger Architecture
+## The Interledger Model
 
-All electronic transfers have to be recorded in stateful systems. Otherwise the same instance of an asset could be sent to two different destinations, essentially duplicating the asset. This is also known as a double-spend. We call these stateful systems *ledgers*.
+[diagram]
```

Comment:
We need a diagram here that ideally shows a sender, 2 connectors, 3 ledgers, and a receiver